src/Commands/field/FieldCreateCommands.php

```diff
@@ -469,16 +447,15 @@ protected function createFieldDisplay(string $context): void
             $values['type'] = $fieldWidget;
         }
 
-        // Command files may customize $values as desired.
-        $handlers = $this->getCustomEventHandlers(sprintf('field-create-%s-display', $context));
-        foreach ($handlers as $handler) {
-            $handler($values);
-        }
+        // Event subscribers may customize $values as desired.
+        $event = new FieldCreateEntityDisplayValuesEvent($context, $values, $this->input, $this->output);
+        $this->eventDispatcher->dispatch($event);
+        $values = $event->getValues();
 
         $storage = $this->getEntityDisplay($context);
 
         if (!$storage instanceof EntityDisplayInterface) {
-            $this->logger()->info(
+            $this->io()->info(
```

Comment:
FYI I only changed success messages to io(). The others have remained logger messages. I'm happy to do whatever you prefer.